bitrise/bitrise.go

```diff
@@ -25,6 +25,31 @@ type Build struct {
 	OriginalBuildParams json.RawMessage `json:"original_build_params"`
 }
 
+// IsRunning ...
+func (b Build) IsRunning() bool {
```

Comment:
The linter's complaining about these being named `b` instead of `build` like below. Wanna just switch them to `build` here?

Reply:
Actually I was given access now 🎉